draft-fossati-tls-attestation.md

```diff
@@ -812,10 +812,10 @@ In this example, a confidential workload is executed on computational
 resources hosted at a cloud service provider.  This is a typical
 scenario for secure, privacy-preserving multiparty computation,
 including anti-money laundering, drug development in healthcare, contact
```

Comment:
```suggestion
resources hosted at a cloud service provider.  This is a typical
scenario that requires secure, privacy-preserving multiparty computation,
environment for the workload. Typical workload examples include anti-money laundering,
drug development in healthcare, contact tracing in pandemic times,
Machine Learning (ML) model training, etc.
```

Reply:
I am not sure the suggested text is correct.  In particular: _"This is a typical
scenario that requires secure, privacy-preserving multiparty computation,
environment for the workload."_ seems backwards.